src/history/hs.c

```diff
@@ -497,7 +497,7 @@ __hs_insert_record(WT_SESSION_IMPL *session, WT_CURSOR *cursor, const uint32_t b
 
     /* Set the current update start time pair as the commit time pair to the history store record.
```

Comment:
The comment should either be one line with the comment close at the end, or the `/*` should be on a line by itself.

Reply:
Fixed.